content/articles/how-to-create-horizontal-scrolling-menu-using-jquery-and-php/index.md

```diff
@@ -0,0 +1,250 @@
+   <h1>Dummy text</h1> quo optio illum hic quia incidunt deleniti! Vitae ut rerum soluta excepturi iusto error, minima facere magnam ipsam repudiandae nesciunt reprehenderit voluptate. Fuga aliquid voluptate nesciunt deserunt! Voluptatibus dicta quae soluta distinctio laudantium ut voluptatem id optio, neque laboriosam inventore, iure dolore quas consequuntur recusandae quasi ex beatae nobis cupiditate! Perspiciatis delectus quos quo assumenda et nobis? In ea voluptas quo accusamus totam nobis, ut tenetur magnam neque commodi ex laudantium at, quas possimus debitis, corrupti dolorem? Laboriosam voluptas, quam cum consequatur, iure reprehenderit eligendi exercitationem ex animi, magni placeat! Quidem libero quas eligendi, exercitationem illo quia. Nam ipsam a ratione quae, delectus autem similique! Voluptatibus quidem aut delectus magnam voluptate eum qui cupiditate voluptas earum. Inventore dolore nulla ut eveniet ullam repellendus necessitatibus in explicabo architecto, sequi magni dolores eius excepturi natus accusamus a est dignissimos sed modi cupiditate aspernatur ipsum, veniam itaque vero. Dicta adipisci dignissimos, consectetur doloribus fugit quae iste repudiandae molestias tempora?</p

```

Comment:
Hello, I'd love to review your article, but this dummy text is just too much,,,and again I'm not sure why you're closing <h1> with </p(not mistaken). For us to review your article, ensure you remove these dummy texts to at most 2 lines.Also ensure to explain every single code snippet.

Reply:
Okay